recipes/samtools/1.16.1/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.16.1" %} # reset build to zero on changing
+{% set htslib_version = "1.16" %}
+
+package:
+  name: samtools-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://github.com/samtools/samtools
+  license: MIT
+  summary: C library for high-throughput sequencing data formats.
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/samtools/samtools.git
+  git_rev: 1.16.1
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - autoconf
+    - automake
+    - make
+    - perl
+  host:
+    - libhts-dev =={{ htslib_version }}
+    - libz-dev
+  run:
+    - libhts =={{ htslib_version }}
+    - libz
+
+outputs:
+  - name: samtools
+    version: {{ version }}
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - autoconf
+        - automake
+        - make
+        - perl
+      host:
+        - libhts-dev =={{ htslib_version }}
+        - libz-dev
+      run:
+        - libhts =={{ htslib_version }}
+        - libz
+    files:
+      - bin/ace2sam
+      - bin/maq2sam-*
+      - bin/md5*
+      - bin/plot-ampliconstats
+      - bin/plot-bamstats
+      - bin/samtools
+      - bin/wgsim
+      - bin/*.pl
+      - bin/*.py
+      - share/man/man1/samtools*.1
+      - share/man/man1/wgsim.1
+    test:
+      commands:
+        - echo -e '@HD\tVN:1.0\tSO:unsorted' | samtools view
+        - samtools --help
+        - samtools --help | grep Version:\ 1.16.1\ \(
```

Comment:
Could you use a version placeholder, instead of a literal? i.e. `{{ version }}` so that it's defined in one place only.